workflowproj/operator.go

```diff
@@ -75,9 +75,9 @@ func GetWorkflowManagedPropertiesConfigMapName(workflow *operatorapi.SonataFlow)
 
 // GetManagedPropertiesFileName gets the default ConfigMap name that holds the managed application property for the given workflow
 func GetManagedPropertiesFileName(workflow *operatorapi.SonataFlow) string {
-	profile := metadata.ProdProfile
+	profile := metadata.QuarkusProdProfile
```

Comment:
So here we have kind of naming separation, by one side we might be executing the SFW "Preview" profile, but, we create an application-prod.prperties because the quarkus profiling?

Reply:
Yes, `prod` is the default Quarkus profiling name: https://quarkus.io/guides/config-reference#default-profiles

Reply:
SonataFlow Preview profile uses Quarkus prod profile. We are not mimicking/exposing Quarkus profiles.

Reply:
yep, I know that "prod" is the default quarkus profile. My question was more related to the fact that when users works with the "Dev" profile, they'll see "application-dev", but, when they work with "Preview" profile, they'll see "application-prod".  Np, but I think this must be clarified in the docs maybe.

Reply:
will do thanks!